openslides_backend/services/datastore/adapter.py

```diff
@@ -182,7 +183,7 @@ def filter(
         collection: Collection,
         filter: Filter,
         mapped_fields: List[str] = None,
-        get_deleted_models: DeletedModelsBehaviour = DeletedModelsBehaviour.NO_DELETED,
+        get_deleted_models: DeletedModelsBehaviour = None,
```

Comment:
Please double check if this change is correct or a misstake.

Reply:
Yes, this is intentional: The way it is coded, the default is `NO_DELETED` anyway, and this change keeps it consistent with the other methods.